troubleshoot/deployments/cloud-enterprise/rebuilding-broken-zookeeper-quorum.md

```diff
@@ -67,29 +67,46 @@ Perform the following steps on each host to back up the Zookeeper data directory
 
 ## Determine the Zookeeper leader [ece_determine_the_zookeeper_leader] 
 
-If a Zookeeper quorum is broken, you must establish the best Zookeeper leader to use for recovery before you start the recovery proces.
+If a Zookeeper quorum is broken, it's crucial to identify the best Zookeeper leader candidate to use for recovery before you start the recovery proces.
 
-The simplest way to check is using the [Zookeeper sync status](verify-zookeeper-sync-status.md) command.
+Collect the following Essential Information (A) and (B) from **all ECE director hosts that have ZK containers running**. This includes any recently created or decommissioned hosts.  Then reach out to [Elastic Support](/troubleshoot/index.md#contact-us).
 
-If this command is not reporting any leaders, then perform the following actions on each director host:
+**Essential Information (A) - Output of file list and sizes of Zookeeper directories**
 
-1. SSH into the host.
-2. Enter the Docker `frc-zookeeper-servers-zookeeper` container and check its `/app/logs/zookeeper.log` logs for `LEADING`:
+```
+# collect disk usage
+find /mnt/data/elastic/*/services/zookeeper/data/ -print -exec du -hs {} \;
+# collect file status
+find /mnt/data/elastic/*/services/zookeeper/data/ -print -exec stat {} \;
+```
 
-    ```sh
-    $ docker exec -it frc-zookeeper-servers-zookeeper bash
-    root@XXXXX:/# cat /app/logs/zookeeper.log | grep 'LEADING'
-    ```
+**Essential Information (B) - ECE Diagnostics**
 
-    This command will return results similar to the following:
+Follow [Run ECE diagnostics tool](run-ece-diagnostics-tool.md) to collect ECE diagnostics. 
```

Comment:
we can leave the title empty and let v3 fill in the title for us (keeps it up to date always)

```suggestion
Follow [](run-ece-diagnostics-tool.md) to collect ECE diagnostics. 
```